db/migrate/20171114183148_profiles.rb

```diff
@@ -0,0 +1,16 @@
+Sequel.migration do
+
+  up do
+    create_table :profiles do
+      column :id, "uuid", :default=>Sequel::LiteralString.new("uuid_generate_v4()"),
```

Comment:
Why not use the usual `uuid` generator?
 
```ruby
Uuid :id, primary_key: true, null: false, unique: false, default: 'uuid_generate_v4()'.lit
```

Looking at the latest migrations and following those conventions is best.

Reply:
Good to know in the future, but same concern as in `db/migrate/20171114183148_profiles.rb` - is it a good idea to change already applied migration scripts?

Reply:
Reviews should happen before the code is deployed :)

Agreed that it's late now

Reply:
We actually did review in 4 separate pulls on bloomberg/cartodb - I think you were just not added as a reviewer at the time.

Reply:
That's great! 

Then don't request my review on this massive PR please, or be explicit about *what* you want me to review. No point in me investing time in this if it's already reviewed. I'd love to be a reviewer, but you need to ping me before the code is merged in order for the review to be as helpful as possible.